garden-backend-service/src/api/schemas/modal/modal_app.py

```diff
@@ -29,3 +35,13 @@ class ModalAppMetadataResponse(ModalAppMetadata):
     @property
     def modal_function_ids(self) -> list[str]:
         return [mf.id for mf in self.modal_functions]
+
+
+class ModalParseMetadataRequest(BaseSchema):
+    file_contents: str
+
+
+class ModalParseMetadataResponse(ModalAppCreateRequest):
```

Comment:
This is a little confusing to me. Could it inherit from "ModalAppMetadata" instead? I think that would get the point across better. It's surprising that this needs to return the owner identity id.

Reply:
yeah that's just as well, no strong preference here